### ISSUE TITLE
Python3 Compatible

### ISSUE THREAD

**Issue by DenisCarriere** [themiurgo/ratelim#6]
I'm getting this indentation errors when I'm testing it in a Python 3.4 env.

I think it's only the latest version of PyPi, your github repo looks good.

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ubuntu/Github/geocoder/geocoder/__init__.py", line 35, in <module>
    from .api import get, yahoo, bing, geonames, google, mapquest
  File "/home/ubuntu/Github/geocoder/geocoder/api.py", line 10, in <module>
    from .google import Google
  File "/home/ubuntu/Github/geocoder/geocoder/google.py", line 4, in <module>
    import ratelim
  File "/home/ubuntu/env/lib/python3.4/site-packages/ratelim/__init__.py", line 18
    self.__max_calls = max_calls
                               ^
TabError: inconsistent use of tabs and spaces in indentation
```

**Comment by themiurgo** [comment]
Just published 0.1.5 on PyPI. Could you check now?

**Comment by DenisCarriere** [comment]
Works fine now! I'm not getting any errors when I'm testing Python 3.4.

Thanks for the quick response!
